editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -199,6 +202,31 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
     }
   }
 
+  // Comment to trigger new pizza build.
```

Comment:
Vultr isn't behaving itself still 😅 

I logged in and killed the instances and DNS records for this PR i.e. `2307` but it seems to be taking a very long time. 

I'll remove this comment and troubleshoot further next week.

Reply:
:eyes: pizza is looking okay now to me, is this resolved or do you have any outstanding vultr bugs/questions?

Reply:
I think it's resolved 😅 

The situation was: 
  - Pizza failed to build
  - I deleted the instances and DNS records for it on Vultr
  - The pizza built but the vultr health check failed. 
  - I pushed a commit
  - The vultr health check passed 🤷 
  
  I'm pretty confident the PR behaves as expected.

Reply:
If you're happy to review @jessicamcinchak I'll merge / update as per feedback 👍

Reply:
A couple tips for future Vultru debugging:
- If only the healthcheck fails, Vultr might just be slow to warm up - wait a couple minutes and try re-running just the failed GH action job to save yourself commits (it usually passes on the next go!)
- Vultr fails to build entirely for a couple common reasons: timeouts (usually around 10m mark), commits pushed close together and an "update" trys to run before the "create" is finished. In these cases, again first trying to re-run just the failed GH action job before doing a full nuke can save a lot of time!